cluster/images/provider-kubernetes/Makefile

```diff
@@ -33,29 +33,34 @@ include ../../../build/makelib/image.mk
 
 img.build:
 ifneq (,$(BUILDX))
-	$(eval CONTAINER_CLI = docker buildx build --platform linux/$(ARCH))
+	$(eval CONTAINER_CLI = docker buildx build --platform linux/$(ARCH) $(BUILDX_ARGS))
+else
+ifneq (,$(shell which podman))
+	$(eval CONTAINER_CLI = podman build --arch $(ARCH) --format docker)
 else
 ifneq (,$(BUILDX_PLUGIN))
-	$(eval CONTAINER_CLI = $(BUILDX_PLUGIN) build --platform linux/$(ARCH))
+	$(eval CONTAINER_CLI = $(BUILDX_PLUGIN) build --platform linux/$(ARCH) $(BUILDX_ARGS))
 else
 	$(eval CONTAINER_CLI = docker build)
 endif
+endif
 endif
 	@$(INFO) $(CONTAINER_CLI) build $(IMAGE)
 	@cp Dockerfile $(IMAGE_TEMP_DIR) || $(FAIL)
-	@cp -R ../../../package $(IMAGE_TEMP_DIR) || $(FAIL)
+	@mkdir $(IMAGE_TEMP_DIR)/package/
+	@cp ../../../package/crossplane.yaml $(IMAGE_TEMP_DIR)/package/crossplane.yaml || $(FAIL)
 	@cd $(IMAGE_TEMP_DIR) && $(SED_CMD) 's|BASEIMAGE|$(OSBASEIMAGE)|g' Dockerfile || $(FAIL)
 	@cd $(IMAGE_TEMP_DIR) && $(SED_CMD) 's|DOCKER_REGISTRY|$(BUILD_REGISTRY)|g' package/crossplane.yaml || $(FAIL)
 	@cd $(IMAGE_TEMP_DIR) && $(SED_CMD) 's|VERSION|$(VERSION)|g' package/crossplane.yaml || $(FAIL)
 	@cd $(IMAGE_TEMP_DIR) && find package -type f -name '*.yaml' -exec cat {} >> 'package.yaml' \; -exec printf '\n---\n' \; || $(FAIL)
-	@$(CONTAINER_CLI) $(BUILDX_ARGS) $(BUILD_ARGS) \
+	@$(CONTAINER_CLI) $(BUILD_ARGS) \
 		--build-arg ARCH=$(ARCH) \
 		--build-arg TINI_VERSION=$(TINI_VERSION) \
 		-t $(IMAGE) \
 		$(IMAGE_TEMP_DIR) || $(FAIL)
-	@$(CONTAINER_CLI) $(BUILDX_ARGS) $(BUILD_ARGS) \
+	@$(CONTAINER_CLI) $(BUILD_ARGS) \
```

Comment:
Same as in controller, shouldn't `latest` be multi-arch?

Reply:
resolved